docs/docs/tutorials/adding_annotations.md

```diff
@@ -100,76 +100,76 @@ $ protoc -I ./proto \
 
 This should generate a `*.gw.pb.go` file.
 
-We also need to add and serve the gRPC-gateway mux in our `main.go` file.
+We also need to add and serve the gRPC-Gateway mux in our `main.go` file.
 
 ```go
```

Comment:
Go code is formatted using tabs. Let's not switch it to using spaces in our documentation.